index.html

```diff
@@ -2543,18 +2588,271 @@ <h1>
+The <a>DID URL dereferencer</a> MUST perform any additional processing indicated
+by the <a>DID URL</a>, such as resolving a fragment into an element. The process
+for completing dereferencing is determined by the mime-type of the <a>DID document</a>
+returned from the <a>DID resolution</a> process. If no additional processing is
+required by the <a>DID URL</a>, the <a>DID document</a> is returned as the 
+<code>content</code>.
+        </p>
+        
+        <p>
+The <code>DID dereference metadata</code> is returned as a map of key-value string pairs as described in
+<a href="#metadata-structure"></a>. This component is REQUIRED and MUST NOT be empty.
+This metadata contains information about the results of
+the dereferencing process. This metadata typically changes between invocations of the <a>DID URL dereferencing</a>
+function. The keys and possible values for this metadata are defined by a registry (TBD). This
+specification defines the following keys and values:
+
+            <dl>
+                <dt>content-type</dt>
+                <dd>The mime-type of the returned content if successful.</dd>
+
+                <dt>error</dt>
+                <dd>The error code from the dereferencing process. The <a>DID resolver</a> MUST supply
+                    this value when there is an error. The possible values
+                    this field are defined by a registry (TBD). This specification defines the following
+                    error values:
+                    
+                    <dl>
+                        <dt>invalid-url</dt>
+                        <dd>The <a>DID URL</a> supplied to the <a>DID URL dereferencing</a> function does not
+                            conform to valid syntax. (See <a href="#did-url-syntax"></a>.)</dd>
+                    </dl>
+                    
+                </dd>
+            
+            </dl>
+        </p>
+        
+        <p>
+The <code>content</code> of this function's output is a byte stream of data. This byte stream MAY 
+represent a <a>DID document</a> in a conformant representation, a portion of a <a>DID document</a>
+in a conformant representation, the results of calling a service
+endpoint, or something else determined by the <a>DID method</a> and its services.
+        </p>
+        
+        <p>
+If the returned <code>content</code> is a <a>DID document</a>, the <a>DID URL dereferencer</a> MAY
+return an additional set of DID document metadata, as described in <a href="#did-resolution"></a>.
+        </p>
+        
+    </section>
+
+    <section>
+        <h2>
+Metadata structure
+        </h2>
+
+        <div class="issue" data-number="58">
+There needs to be a registry established for handling input options and rules defined for 
+how to handle an unknown input option. A similar issue applies for all metadata structures
+returned from these functions. 
+        </div>
+        
+        <p>
+Metadata structures are maps of key-value string pairs. All keys within this map
```

Comment:
```suggestion
Metadata structures are maps of property-value string pairs. All properties within this map
```

Reply:
Either property-value or name-value... prefer property-value as that's what's used elsewhere... forget the issue number, but this is being actively discussed.

Reply:
issue #204 for anyone reading and looking for reference.

Reply:
Will leave as is pending resolution of #204 which can change it to be consistent through the document.